shipment/api/let_me_ship.py

```
@@ -286,16 +346,16 @@ def create_letmeship_shipment(
             },
             'goodsValue': value_of_goods,
             'parcelList': parcel_list,
-            'pickupInterval': {'date': pickup_date},
+            'pickupInterval': pickup_interval,
             'contentDescription': description_of_content,
```

Ruff (0.11.9)

Dictionary key literal `'contentDescription'` repeated

Remove repeated key literal `'contentDescription'`

(F601)